src/PrestaShopBundle/Form/Admin/Type/TranslatableType.php

```diff
@@ -129,7 +136,13 @@ public function configureOptions(OptionsResolver $resolver)
         $resolver->setDefaults([
             'type' => TextType::class,
             'options' => [],
-            'locales' => $this->locales,
+            'only_enabled_locales' => true,
```

Comment:
I've added this option to keep compatibility with inputs where actually only enabled languages can be used, but now I think if we have this kind of use case at all?

Reply:
If the usecase does not exist, what would the code look like ? You modify `$locales` directly ?

Reply:
If we don't have use case for displaying only active languages, then I'd only inject all locales and remove this config.

Reply:
or you can set it to false by default

Reply:
I can see at least one use case in Manufacturer form, we only want to write description and short description for activated languages.

Reply:
> we only want to write description and short description for activated languages

Why is it so? As a merchant if you install new language and want to keep it disabled until you prepare content for, it doesn't make sense, does it? :thinking:

Reply:
I didn't think about this use case indeed! I thought you want to focus on activated languages and not be bothered by useless ones.
But you can still uninstall them if you really don't want to use them.